### DISCUSSION TITLE
faces

### DISCUSSION THREAD

**Discussion by CasualSpec** [trickerer/Trinity-Bots#202] [Q&A]
why are the bot faces look odd https://ibb.co/qrJJL7T

**Comment by trickerer** [comment]
What client is used? Do you have any client modifications (patch-4+.mpq, etc...)?

This sometimes happens to people using backported HD textures, but I don't see that here.

**Comment by zaicopx** [comment]
> why are the bot faces look odd https://ibb.co/qrJJL7T

The problem is your hd texture. Use this HD texture, no glitch for npcbot face.

https://www.mediafire.com/file/y4mzsjcezxa7idk/TextureHD+Wotlk+(Full)(Update+5.2.1).7z/file
